templates/kotsadm-config.yaml

```diff
@@ -5,7 +5,8 @@ metadata:
     {{- include "admin-console.labels" . | nindent 4 }}
   name: kotsadm-confg
 data:
-  initial-app-images-pushed: "false"
+  initial-app-images-pushed: "{{ .Values.initialAppImagesPushed | quote }}"
+  force-original-image-names: "{{ .Values.forceOriginalImageNames | quote }}"
```

Comment:
this is a new param that I'll need to have kots read, so that it doesn't require a registry spec in airgap